api/v1alpha1/eraserconfig_types.go

```diff
@@ -91,15 +91,16 @@ type ContainerConfig struct {
 }
 
 type ManagerConfig struct {
-	Runtime           Runtime          `json:"runtime,omitempty"`
-	OTLPEndpoint      string           `json:"otlpEndpoint,omitempty"`
-	LogLevel          string           `json:"logLevel,omitempty"`
-	Scheduling        ScheduleConfig   `json:"scheduling,omitempty"`
-	Profile           ProfileConfig    `json:"profile,omitempty"`
-	ImageJob          ImageJobConfig   `json:"imageJob,omitempty"`
-	PullSecrets       []string         `json:"pullSecrets,omitempty"`
-	NodeFilter        NodeFilterConfig `json:"nodeFilter,omitempty"`
-	PriorityClassName string           `json:"priorityClassName,omitempty"`
+	Runtime             Runtime           `json:"runtime,omitempty"`
+	OTLPEndpoint        string            `json:"otlpEndpoint,omitempty"`
+	LogLevel            string            `json:"logLevel,omitempty"`
+	Scheduling          ScheduleConfig    `json:"scheduling,omitempty"`
+	Profile             ProfileConfig     `json:"profile,omitempty"`
+	ImageJob            ImageJobConfig    `json:"imageJob,omitempty"`
+	PullSecrets         []string          `json:"pullSecrets,omitempty"`
+	NodeFilter          NodeFilterConfig  `json:"nodeFilter,omitempty"`
+	PriorityClassName   string            `json:"priorityClassName,omitempty"`
+	AdditionalPodLabels map[string]string `json:"additionalPodLabels,omitempty"`
```

Comment:
Please remove this change, as well as the one to `api/v1alpha2/eraserconfig_types.go` (and any autogenerated files that follow from those). We'd like to leave those unchanged. `unversioned` should then be identical to `v1alpha3`.

Reply:
I have also committed the manifest_staging directory now.

Also, I have removed the changes to api/v1alpha1 and api/v1alpha2 and ran the autogeneration again.

Was it correct that I also removed my changes to api/v1alpha1/config/config.go and api/v1alpha2/config/config.go? You didn't mention these explicitly, but I assume you want to keep everything stable in api/v1alpha1 and api/v1alpha2?

Reply:
Yes, that's correct. I'll approve the CI run now